sources/Interop/D3D12/um/d3dx12/D3D12.Manual.cs

```diff
@@ -23,6 +23,43 @@ public static unsafe partial class D3D12
             Count = 1,
             Quality = 0,
         };
+        
+        [return: NativeTypeName("HRESULT")]
+        public static int CreateRootSignature([NativeTypeName("ID3D12Device*")] ID3D12Device* device, [NativeTypeName("const D3D12_ROOT_SIGNATURE_DESC*")] D3D12_ROOT_SIGNATURE_DESC* rootSignatureDesc, [NativeTypeName("ID3D12RootSignature**")] ID3D12RootSignature** rootSignature)
+        {
+            ID3DBlob* pSignature = default;
+            ID3DBlob* pError = default;
+            
+            try
```

Comment:
Why is the `try` needed? The native code doesn't have one and we aren't throwing an exception here, in fact we are explicitly not throwing and returning the `HRESULT`

Reply:
I'll remove, I just saw the ComPtr and blindly added a finally as equivalent to the `Microsoft::WRL::ComPtr` dtor